src/main/webapp/app/course/manage/course-detail.component.html

```diff
@@ -50,7 +50,7 @@ <h4 class="modal-title"><span [jhiTranslate]="'artemisApp.course.archive.popup.t
                     <div class="modal-body">
                         <p [jhiTranslate]="'artemisApp.course.archive.popup.question'" [translateValues]="{ title: course.title }">Are you sure you want to archive the course?</p>
                         <p [jhiTranslate]="'artemisApp.course.archive.popup.statement1'">
-                            The process will compress all student code repositories, file upload exercises, modeling exercises, and text exercises for exercises and exams.
+                            The process will compggggress all student code repositories, file upload exercises, modeling exercises, and text exercises for exercises and exams.
```

Comment:
I think this is a typo? :D

Reply:
lol yes
```suggestion
                            The process will compress all student code repositories, file upload exercises, modeling exercises, and text exercises for exercises and exams.
```
I guess my 'g' key got stuck